language_formatters_pre_commit_hooks/utils.py

```diff
@@ -9,6 +10,8 @@
 
 import requests
 
+CHECKSUM_CHUNK_SIZE_KiB = 64 * 1024
```

Comment:
This constant should be kept private (`_`-prefixed) or defined within `does_checksum_match`

Reply:
I think I will get rid of the chunking since the `.jar` is <5 MB (not that much memory)